### PR TITLE
Release 2.6

### PR DESCRIPTION
**Pull request by DavertMik**
Bringing all changes + adding documentation

### CODE REVIEW

File: runio.js [hunk 1]
```diff
@@ -305,22 +305,20 @@ title: ${name}
...
     const dir = 'website';
-    await git((fn) => {
-      if (!fs.existsSync(dir)) {
-        fn.clone('git@github.com:codecept-js/website.git', dir);
-      } else {
-        fn.pull();
-      }
-    });
+    if (fs.existsSync(dir)) {
+      await exec('rm -rf ' + dir);
```
Comment (DavertMik) [inline runio.js]: Codacy Issue found: [Unexpected string concatenation.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Clones added
============
- test/runner/comment_step_test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126&bid=17245758)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- test/helper/Playwright_test.js  3
- test/runner/comment_step_test.js  1
- lib/helper/WebDriver.js  22
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126&bid=17245758)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- test/helper/Playwright_test.js  3
- lib/helper/WebDriver.js  22
- test/runner/comment_step_test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126&bid=17245758)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- test/runner/comment_step_test.js  1
- lib/helper/WebDriver.js  22
- test/helper/Playwright_test.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126&bid=17245758)

**Comment by gkushang** [comment]
@DavertMik should the below PR (shadow dom support) go into this release too?

https://github.com/Codeception/CodeceptJS/pull/2276

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- test/runner/comment_step_test.js  1
- test/helper/Playwright_test.js  2
- lib/helper/WebDriver.js  22
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126&bid=17245758)

**Comment by DavertMik** [comment]
@gkushang Yes! I am writing release notes for shadow dom support right now

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- test/helper/Playwright_test.js  2
- lib/helper/WebDriver.js  22
- test/runner/comment_step_test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126&bid=17245758)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/WebDriver.js  22
- test/helper/Playwright_test.js  2
- test/runner/comment_step_test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126&bid=17245758)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- test/runner/comment_step_test.js  1
- test/helper/Playwright_test.js  2
- lib/helper/WebDriver.js  22
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126&bid=17245758)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- test/helper/Playwright_test.js  2
- lib/helper/WebDriver.js  22
- test/runner/comment_step_test.js  1
- lib/helper/Puppeteer.js  1
- lib/helper/Playwright.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126&bid=17245758)

**Comment by pablopaul** [comment]
@DavertMik I merged #2302 into this branch, please add it to the changelog, thanks

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- test/helper/Playwright_test.js  1
- lib/helper/Puppeteer.js  1
- lib/helper/Playwright.js  1
- lib/helper/WebDriver.js  22
- test/runner/comment_step_test.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126&bid=17245758)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- lib/helper/WebDriver.js  22
- lib/helper/Puppeteer.js  1
- test/runner/comment_step_test.js  1
- test/helper/Playwright_test.js  1
- lib/helper/Playwright.js  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5264126&bid=17245758)

**Pull request merged**